docs/sidebars.js

```diff
@@ -1,109 +1,111 @@
-  // tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
-  // But you can create a sidebar manually
   sidebarJSBolt: [
     {
-      type: 'doc',
-      id: 'index', // document ID
-      label: 'Bolt for JavaScript', // sidebar label
-      className: 'sidebar-title',
+      type: "doc",
+      id: "index",
+      label: "Bolt for JavaScript",
+      className: "sidebar-title",
+    },
+    "getting-started",
+    {
+      type: "category",
+      label: "Slack API calls",
+      items: ["concepts/message-sending", "concepts/web-api"],
+    },
+    {
+      type: "category",
+      label: "Events",
+      items: ["concepts/message-listening", "concepts/event-listening"],
+    },
+    {
+      type: "category",
+      label: "Interactivity & Shortcuts",
+      items: [
+        "concepts/acknowledge",
+        "concepts/shortcuts",
+        "concepts/commands",
+        "concepts/action-listening",
+        "concepts/action-respond",
+        "concepts/creating-modals",
+        "concepts/updating-pushing-views",
+        "concepts/view-submissions",
+        "concepts/options",
+        "concepts/publishing-views",
+      ],
+    },
+    "concepts/assistant",
+    "concepts/custom-steps",
+    {
+      type: "category",
+      label: "App Configuration",
+      items: [
+        "concepts/socket-mode",
+        "concepts/error-handling",
+        "concepts/logging",
+        "concepts/custom-routes",
+        "concepts/deferring-initialization",
+        "concepts/receiver",
+      ],
     },
-    'getting-started',
     {
-      type: 'category',
-      label: 'Basic concepts',
+      type: "category",
+      label: "Middleware & Context",
       items: [
-        'basic/message-listening',
-        'basic/message-sending',
-        'basic/event-listening',
-        'basic/web-api',
-        'basic/action-listening',
-        'basic/action-respond',
-        'basic/acknowledge',
-        'basic/shortcuts',
-        'basic/commands',
-        'basic/creating-modals',
-        'basic/updating-pushing-views',
-        'basic/view-submissions',
-        'basic/publishing-views',
-        'basic/custom-steps',
-        'basic/options',
-        'basic/authenticating-oauth',
-        'basic/socket-mode',
+        "concepts/global-middleware",
+        "concepts/listener-middleware",
+        "concepts/context",
       ],
     },
     {
-      type: 'category',
-      label: 'Advanced concepts',
+      type: "category",
+      label: "Authorization & Security",
       items: [
-        'advanced/error-handling',
-        'advanced/authorization',
-        'advanced/token-rotation',
-        'advanced/conversation-store',
-        'advanced/global-middleware',
-        'advanced/listener-middleware',
-        'advanced/context',
-        'advanced/deferring-initialization',
-        'advanced/logging',
-        'advanced/custom-routes',
-        'advanced/receiver',
+        "concepts/authorization",
```

Comment:
I'd put the authorization article _after_ the authenticating-with-OAuth one, since the latter is a pre-req for using the former.